website/intro/getting-started/build.html.md

```diff
@@ -70,11 +68,6 @@ with access to a default VPC. For EC2 Classic users, please use `t1.micro` for
 `us-east-1` then you will need to choose an AMI in that region
 as AMI IDs are region specific.
 
-Replace the `ACCESS_KEY_HERE` and `SECRET_KEY_HERE` with your
-AWS access key and secret key, available from
-[this page](https://console.aws.amazon.com/iam/home?#security_credential).
-We're hardcoding them for now, but will extract these into
-variables later in the getting started guide.
 
 ~> **Note**: If you simply leave out AWS credentials, Terraform will
```

Comment:
I think this Note could be rephrased since we're removing the credentials from the provider block.